soc/arm/microchip_mec/mec172x/reg/mec_defs.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Microchip Technology Inc. and its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _MEC_DEFS_H
+#define _MEC_DEFS_H
+
+#ifndef BIT
```

Comment:
Do we need these, can we not use what is in Zephyr

Reply:
Agreed. I'll remove the file.

Reply:
File is removed as of today's rebase.